docs/book/pagination.md

```diff
@@ -412,13 +412,15 @@ our modules:
   <?php if (isset($this->previous)): ?>
     <li class="page-item">
       <a class="page-link" href="<?= $this->url($this->route, [], ['query' => ['page' => $this->previous]]) ?>">
-        &lt;&lt;
+      <span aria-hidden="true">&laquo;</span>
+        <span class="sr-only">Previous</span>
       </a>
     </li>
   <?php else: ?>
     <li class="page-item disabled">
       <a class="page-link" href="#">
-        &lt;&lt;
+      <span aria-hidden="true">&laquo;</span>
+        <span class="sr-only">Previous</span>
       </a>
```

Comment:
The `a` element with all children can be replaced with one `span` element:

```suggestion
      <span class="page-link">Previous</span>
```

https://getbootstrap.com/docs/4.4/components/pagination/#disabled-and-active-states